src/applications/vre/28-1900/config/chapters/communication-preferences/communicationPreferences.js

```diff
@@ -80,18 +80,27 @@ export const uiSchema = {
       'ui:title': 'Mornings 6:00 to 10:00 a.m.',
       'ui:options': {
         hideEmptyValueInReview: true,
+        widgetProps: {
+          false: { 'aria-describedby': 'appointment-time-desc' },
```

Comment:
I want to validate with @AngelaFowler82 quickly if attaching `aria-describedby` on each radio option would be too verbose, or if it would be OK to attach to just the first radio option. Will get a response back asap!

Reply:
Sounds good, thanks Josh!

Reply:
It's an individually rendered checkbox, not a group. There is a group checkbox component in the component library, but it isn't being used inside the form system

Reply:
Got it! Thanks Mottie! I just checked in with Angela, and she said `aria-describedby` should be attached to all the checkboxes _if_ the added information is really important, otherwise it can get verbose.

This is a bit tricky, since the description hints to folks that their first selection may not get picked. I wonder if we can do the following:

If it is very important:
- Attach `aria-describedby` to all checkboxes
- Shorten the content in the `aria-describedby` 

If it is good to know, but not very important:
- Attach `aria-describedby` just to the first checkbox

Reply:
This feels like it falls more into the category of it's good to know, but not very important. @joshkimux are you fine with just attaching it to the first checkbox?

Reply:
Yes, let's do that! 💯